junkie/core_context.py

```diff
@@ -79,7 +79,9 @@ def _call(self, factory_func: Callable, stack: ExitStack, instance_name: str):
                 arg = self._call(self._factories[name], stack, name)
 
             else:
-                raise Exception("Not found: " + name)
+                raise Exception(
+                    "Unable to find argument '{}' for factory function '{}'".format(name, instance_name)
```

Comment:
As it is for beginners, maybe this might be a little bit confusing as soon as you passing a class as a „factory“...

Reply:
`MissingDependencyError: Unable to find argument '{}' for factory function '{}'`